woocommerce/woocommerce
# Update woocommerce-admin package to 2.1.5

This bumps `woocommerce/woocommerce-admin` in Composer to the latest published package, version 2.1.5.
 
## Testing Instructions
 
Via https://github.com/woocommerce/woocommerce-admin/pull/6730

- On a new store, go to the store onboarding flow and use one of these countries -> AU, CA, GB, IE and NZ
- Continue till the free business features step, WC Pay should not be part of that list.
 
## Changelog
 
- Tweak: Set International Country Feature Flag off
 
### Changelog entry
 
> Update - WooCommerce Admin package 2.1.5
